extensions/cornerstone-dicom-seg/src/getToolbarModule.ts

```diff
@@ -118,5 +120,22 @@ export function getToolbarModule({ servicesManager }: withAppTypes) {
         };
       },
     },
+    {
+      name: 'evaluate.cornerstone.segmentation.synchronizeDrawingRadius',
+      evaluate: ({ button, radiusOptionId }) => {
+        const toolGroupIds = toolGroupService.getToolGroupIds();
+        if (!toolGroupIds?.length) {
+          return;
+        }
+
+        for (const toolGroupId of toolGroupIds) {
+          const brushSize = cstUtils.segmentation.getBrushSizeForToolGroup(toolGroupId);
+
+          if (brushSize) {
+            button.props.options.find(option => option.id === radiusOptionId).value = brushSize;
```

Comment:
I think we should consider adding a `getOptionById` method to the ToolbarService to do the find here. Likely a useful utility.